tensorflow_addons/losses/contrastive.py

```diff
@@ -88,17 +92,19 @@ class ContrastiveLoss(tf.keras.losses.Loss):
 
     def __init__(
         self,
-        margin=1.0,
-        reduction=tf.keras.losses.Reduction.SUM_OVER_BATCH_SIZE,
-        name="contrasitve_loss",
-    ):
+        margin: Number = 1.0,
+        reduction: tf.keras.losses.Reduction = tf.keras.losses.Reduction.SUM_OVER_BATCH_SIZE,
+        name:str = "contrasitve_loss",
+    ) -> None:
```

Comment:
When using `__init__` you don't have to specify the return type because it's obvious.